docs/site/examples.md

```diff
@@ -33,6 +33,15 @@ examples:
     - language: powershell
       url: "/tree/master/examples/knative/powershell/kn-ps-email"
 
+  - title: Host Maintenance Alarm Actions
+    usecases:
+    - item: notification
+    id: kn-ps-slack-function
```

Comment:
is the id right?

Reply:
No. Fixed.

ID doesn't seem to have any impact on the page.